zulipterminal/scripts/render_symbols.py

```diff
@@ -0,0 +1,42 @@
+import sys
+
+import urwid
+
+from zulipterminal.config import symbols
+
+
+palette = [
+    ('header', 'white, bold', 'light blue'),
+    ('footer', 'black', 'white'),
+]
+
+variables = vars(symbols)
+symbol_dict = {symbol: variables[symbol] for symbol in variables
+               if not symbol.startswith('__') and not symbol.endswith('__')}
+
+symbols_list = [urwid.Columns([urwid.Text(symbol, align='center'),
+                               urwid.Text(symbol_dict[symbol], align='center')
+                               ]) for symbol in symbol_dict]
```

Comment:
This is sort of hacky, but we may specify `align='right'` and `'left'` to respective `Text` widgets and provide some `dividechars` to `Column`, if we want to reduce the distance between name and symbol?

Reply:
The `LineBox` however would still remain wide - as far as I tested :(

Reply:
Yeah, the width seems to vary based on the font size and the width allowed to the terminal. I've tried reducing that using a padding, but Neil suggested to just keep it simple and not worry about the width. The current iteration would use the complete width available since there is no padding.

Reply:
With no padding, each symbol name should render in a single line in most terminal font size and width combinations, beyond which it might not be feasible for us to consider.